features/FxA-28-change-email/README.md

```diff
@@ -0,0 +1,53 @@
+
+Change Email
+======================
+
+https://mozilla.aha.io/features/FXA-28
+
+As an FxA user when I lose control of the email account  that is my username in Firefox Accounts I want to be able to change that FxA email address so that I don't lose access to my data and other FxA-relying services.
+* When a user changes their email in Firefox Accounts, it simply updates in other clients and relying services like display name without requiring them to reconnect.
```

Comment:
It's not clear what "like display name" means here and maybe we can just remove it.  IIUC, what we're trying to capture here is that it just looks like part of my profile data has changed, not that I'm getting a new account.

We should consider what this will look like from the POV of a relying service.  Should we try to push them some sort of notification that says the email has changed?  Do we have this as a hard requirement from any existing services e.g. pocket or basket?